src/style.css

```diff
@@ -0,0 +1,72 @@
+body {
+  margin: 0;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  display: block;
+  background: url(images/logo.png) no-repeat;
+}
+
+.menu-link {
```

Comment:
check this link on your page, the height is too small, you have to be able to click on the link not only right in the text